test/e2e/features/steps/first_steps.go

```diff
@@ -0,0 +1,83 @@
+package steps
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	// "testing"
+	// "k8s.io/kubernetes/test/e2e"
+	"github.com/DATA-DOG/godog"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/config"
+	"github.com/onsi/ginkgo/reporters"
+	"k8s.io/klog"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+// Suite is an interface that describes the parts of the godog.Suite interface
+// used within the package
+type Suite interface {
+	Step(interface{}, interface{})
+	// BeforeScenario(func(interface{}))
+	// BeforeSuite(func())
+	// AfterSuite(func())
+	// AfterScenario(func(interface{}))
+	// AfterScenario(func(interface{}))
+}
+
+var testingMain *testing.M
+
+func existingTest(existingTestName string) error {
```

Comment:
This method should be able to take the existing test name and call the Ginkgo Framework via m.Run() or similar.

I'm having hard time finding the right entry point.